TABTestKit.xcodeproj/xcshareddata/xcschemes/BDD.xcscheme

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Are all of these schemes still needed if we're not trying to support subspecs with Carthage?